marc_to_solr/lib/traject_config.rb

```diff
@@ -300,7 +300,11 @@
                      else
                        MarcExtractor.cached("950b").extract(record).first
                      end
-    accumulator[0] = Time.parse(cataloged_date).utc.strftime("%Y-%m-%dT%H:%M:%SZ") unless cataloged_date.nil?
+    begin
+      accumulator[0] = Time.parse(cataloged_date).utc.strftime("%Y-%m-%dT%H:%M:%SZ") unless cataloged_date.nil?
+    rescue
+      logger.error "#{record['001']} - error parsing cataloged date #{cataloged_date}"
+    end
```

Comment:
@hackartisan do we want to add 'Failed to index ....and the error'. So that we know that it failed to index.

Reply:
The rest of the errors we log are to inform us about cataloging info.

Reply:
I don't think it failed to index the record. It just failed to get this value. Is that what you mean?

Reply:
oh ok, I thought it failed to index the record.